src/spikeinterface/core/tests/test_basesorting.py

```diff
@@ -144,14 +165,15 @@ def test_npy_sorting():
 
 
 def test_empty_sorting():
-    sorting = NumpySorting.from_dict({}, 30000)
+    sorting = NumpySorting.from_unit_dict({}, 30000)
 
     assert len(sorting.unit_ids) == 0
 
-    spikes = sorting.get_all_spike_trains()
-    assert len(spikes) == 1
-    assert len(spikes[0][0]) == 0
-    assert len(spikes[0][1]) == 0
+    with pytest.warns(DeprecationWarning):
```

Comment:
To be honest, I am oddly suprised that you decided to test deprecation warnings. I think that I am on the side that is likely to be wrong by making too much testing and this is like on the line for me.

Not that I think is wrong, just pondering about my surprise.

Reply:
@samuelgarcia is changing....